internal/config/config_test.go

```diff
@@ -166,25 +167,6 @@ plugins:
 	})
 
 	Context("with invalid keys", func() {
-		It("should return a save error", func() {
```

Comment:
This test added does not work at all. We did not get that because it has not been called. So, I only removed it.

Reply:
Originally `Config.Save()` returned an error when project version was < 3. Now it does not, so this test can be removed.